bin/node-debug.js

```diff
@@ -194,7 +194,19 @@ function openBrowserAndPrintInfo() {
   );
 
   if (!config.options.cli) {
-    open(url);
+    // try to launch the URL in one of those browsers in the defined order
+    // (but if one of them is default browser, then it takes priority)
+    open(url, {
+        preferredBrowsers : ['chrome', 'chromium', 'opera']
+      }, function(err, okMsg) {
```

Comment:
1. misconfiguration (will not happen here)
2. chrome/chromium/opera not found in the system
3. error from browser-launcher, unable to start the executable for some reason (not sure in which conditions this could happen; probably with improper rwx permissions on unixes and maybe some other circumstances)

(look for `new Error` in https://raw.githubusercontent.com/jakub-g/biased-opener/master/index.js)